packages/rx_bloc_test/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: rx_bloc_test
 description: A Flutter package with the goal to enable testing rxBlocs from the FlutterRxBloc package with ease.
-version: 4.0.0
+version: 4.1.0
```

Comment:
When upgrading dev dependencies we do not need to bump the version of the package